FE/angular/src/app/components/movie/movie.component.html

```diff
@@ -1,38 +1,6 @@
 <app-go-back></app-go-back>
-
-<ul *ngIf="movie" class="movie">
-  <li class="grid grid-bleed">
-    <div class="image col-3"><img [src]="movie.Poster" /></div>
-    <div class="content col-9 col-grid">
-      <div class="title">{{ movie.Title }} ({{ movie.Year }})</div>
-      <ul class="metadata">
-        <li class="rated">
-          <div>{{ movie.Rated }}</div>
-        </li>
-        <li class="runtime">{{ movie.Runtime }}</li>
-        <li class="genre">{{ movie.Genre }}</li>
-        <li class="release">{{ movie.Released }}</li>
-      </ul>
-      <ul class="details">
-        <li>
-          <strong>Director:</strong>
-          {{ movie.Director }}
-        </li>
-        <li>
-          <strong>Writer:</strong>
-          {{ movie.Writer }}
-        </li>
-        <li>
-          <strong>Stars:</strong>
-          {{ movie.Actors }}
-        </li>
-      </ul>
-      <div class="plot">
-        {{ movie.Plot }}
-      </div>
-      <div class="nav flex-footer">
-        <app-go-imdb [imdbId]="movie.imdbID"></app-go-imdb>
-      </div>
-    </div>
-  </li>
-</ul>
+<ng-container *ngIf="movieSubscription | async; let movie">
+  <div class="movie">
```

Comment:
Why is this `div` outside the `app-movie-card` ?

Reply:
Instead of UL/LI, I've made the list as component, so div( .movie) is the main container for the list components.